### ISSUE TITLE
Add CartoCSS to the CartoDB Platform doc

### ISSUE THREAD

**Issue by csobier** [CartoDB/docs#330]
@carlostallon Similar to #327 , I need a collaborator resource file structure changes. I have WIP (work-in-progress) files in the branch, add_cartocss_files but really need the structure created before finalizing the edits. What do you need from me?  I have several files/images in the branch.

[ ] Add CartoCSS placeholder in [CartoDB Platform home page](http://docs.cartodb.com/cartodb-platform.html)
[ ] Add CartoCSS to the CartoDB Platform Menu item (top of page)
[ ] Add CartoCSS to the CartoDB Platform home page, left-sidebar

![image](https://cloud.githubusercontent.com/assets/13319925/9501859/d62d35de-4bfb-11e5-9a1c-a7036f629d3c.png)

**Comment by matallo** [comment]
I can take this, but let's discuss in tomorrow catch-up @CartoDB/design

**Comment by csobier** [comment]
Changed milestone to Future Community.  This file structure change cannot be implemented until the Doc is redesigned.  I was instructed that any new content must be inserted into the current structure until we migrate to a new system :-(

**Comment by andrewxhill** [comment]
hey, after our meeting last we, we decided not to stop this right? we are going to push forward without letting the file structure stop us.

**Comment by andrewxhill** [comment]
we should target next thursday for deployment, to alight https://github.com/CartoDB/cartodb-management/issues/4226#issuecomment-141006320

**Comment by csobier** [comment]
Yep, I've got a ton of content that needs review.  Just adding the final comp-op descriptions....  I have this  targeted for the Cabra del Santo Cristo release. (Sept.29th)

**Comment by andrewxhill** [comment]
Okay, that makes sense. Let's assess on Monday how close you are and if it can be accelerated or we are too far off. Thanks!

**Comment by andrewxhill** [comment]
Here is the information for what runs in Torque CartoCSS. 

https://github.com/CartoDB/torque-reference/blob/master/1.0.0/reference.json

It is a bit technical, but we can sit down and parse it together if that would help
